recipe/meta.yaml

```diff
@@ -22,16 +21,19 @@ requirements:
     - setuptools_scm
```

Comment:
You may want to add version info https://github.com/fatiando/pooch/blob/94aaccfac4eb21fa90d392a1bc0c079118a8f873/pyproject.toml#L4.

We are probably ok but to have toml support that it is asking for we need at least setuptools 45.  We are probably just fine though.